src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -37,6 +37,8 @@ const StyledCheckBoxContainer = styled.label`
   align-items: center;
   user-select: none;
   width: fit-content;
+  ${props =>
+    props.width && `width: ${props.width}; justify-content: space-between;`}
```

Comment:
A couple of questions:
1. Are we always assuming that if `width` is provided we'll use justify between layout?
2. The current implementation isn't supporting t-shirt sizing of width such as 'small'/'medium'/... which isn't consistent with grommet `width` functionalities.